msticpy/sectools/tiproviders/kql_base.py

```diff
@@ -309,6 +308,19 @@ def _create_query_provider(self, **kwargs):
         query_provider.connect(connect_str)
         return query_provider
 
+    @staticmethod
+    def _get_spelled_variants(name: str, **kwargs) -> Any:
+        """Return value with matching variant spelling key."""
```

Comment:
Nice!

Reply:
This was a bit of nightmare of my own making. Had 2 different variants of each in the msticpyconfig.yaml example